src/AltinnCore/Designer/Controllers/RulesController.cs

```diff
@@ -16,21 +20,30 @@ public class RulesController : Controller
         private readonly IRepository _repository;
         private readonly ICodeGeneration _codeGeneration;
         private readonly ICompilation _compilation;
+        private readonly ServiceRepositorySettings _settings;
+        private readonly IHttpContextAccessor _httpContextAccessor;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="RulesController"/> class
         /// </summary>
         /// <param name="repositoryService">The service repository service</param>
         /// <param name="codeGenerationService">The code generation service</param>
         /// <param name="compilationService">The service compilation service</param>
+        /// <param name="repositorySettings">The service repository settings</param>
+        /// <param name="httpContextAccessor">The http context accessor</param>
         public RulesController(
             IRepository repositoryService,
             ICodeGeneration codeGenerationService,
-            ICompilation compilationService)
+            ICompilation compilationService,
+            IOptions<ServiceRepositorySettings> repositorySettings,
+            IHttpContextAccessor httpContextAccessor)
         {
             _repository = repositoryService;
             _codeGeneration = codeGenerationService;
             _compilation = compilationService;
+            _settings = repositorySettings.Value;
+            _httpContextAccessor = httpContextAccessor;
+
```

Comment:
Sylecop fails here, remove the black line:  error SA1513: Closing brace should be followed by blank line